diff -urN locust-0.8.org/setup.py locust-0.8/setup.py
--- locust-0.8.org/setup.py	2017-09-19 11:54:44.000000000 +0200
+++ locust-0.8/setup.py	2018-07-07 15:24:44.475173041 +0200
@@ -41,7 +41,7 @@
     packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
     include_package_data=True,
     zip_safe=False,
-    install_requires=["gevent>=1.2.2", "flask>=0.10.1", "requests>=2.9.1", "msgpack-python>=0.4.2", "six>=1.10.0", "pyzmq==15.2.0"],
+    install_requires=["gevent>=1.2.2", "flask>=0.10.1", "requests>=2.9.1", "msgpack>=0.4.2", "six>=1.10.0", "pyzmq>=15.2.0"],
     test_suite="locust.test",
     tests_require=['unittest2', 'mock'],
     entry_points={
